ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -130,6 +130,13 @@ tests:
           cpu: 500m
           memory: 1Gi
     workflow: optional-operators-ci-aws
+- as: security
+  optional: true
```

Comment:
I'm just pumping to ensure that setting this to optional PRs that didn't pass the check won't be a blocker for the merge. that is what we want right?

Reply:
this is totally the intention, yes. The lane should be informational, not blocking.